packages/shared/src/components/auth/common.tsx

```diff
@@ -41,6 +41,7 @@ export const providerMap: ProviderMap = {
     icon: <GoogleIcon className="socialIcon" secondary />,
     provider: 'Google',
     style: { backgroundColor: '#FFFFFF', color: '#0E1217' },
+    className: 'border border-theme-divider-tertiary',
```

Comment:
Is this expected for both light and dark?

Reply:
From the slack thread i read it as they wanted this color (divider-teritary) for both

Reply:
Yep, should be fine indeed, it sorts light/dark itself